include/envoy/access_log/access_log.h

```diff
@@ -109,12 +109,14 @@ class Formatter {
    * @param response_headers supplies the response headers.
    * @param response_trailers supplies the response trailers.
    * @param stream_info supplies the stream info.
+   * @param response_body supplies the response body.
    * @return std::string string containing the complete formatted access log line.
    */
   virtual std::string format(const Http::RequestHeaderMap& request_headers,
                              const Http::ResponseHeaderMap& response_headers,
                              const Http::ResponseTrailerMap& response_trailers,
-                             const StreamInfo::StreamInfo& stream_info) const PURE;
+                             const StreamInfo::StreamInfo& stream_info,
+                             const absl::string_view& response_body) const PURE;
```

Comment:
Two reasons to put `response_body` into access_log_formatter. 

1) To support embedding response_body inside other command operators, such as
text_string: %RESPONSE_CODE%: %RESP_BODY% %REQ(:path)%
or
json_string: 
    code: %RESPONSE_CODE%
    body: %RESP_BODY%
    path: %REQ(:path)%

2) well,  to support writing response_body to access log. 
 it is unlikely due to the size,  But some users may want to do that.

Reply:
The confusion may come form the code is under access_log folder.  Since this code is going to be shared by LocalReply,   we can move access_log_fomatter out of access_log folder, move them into souce/common/common, to be at the same place as SubstitutionFormatString

Reply:
Agreed on moving the code. Can you do that as either a precursor to this PR or as a follow up?

If we leave this as a command operator I think it needs to be documented as such. The problem is that it won't always work for access logging, so seems pretty odd? How do you recommend documenting that?

Reply:
Maybe LOCAL_REPY_BODY or something?

Reply:
I see.  Now I can understand, access_log may never support logging response_body since it is not saved.  I will call this a local_reply_body for now, but the command operator is renamed to %LOCAL_REPLY_BODY%

I will use a follow up PR to move the access_log_formatter code

Reply:
For documentation,   I will update this [link](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage.html?highlight=command%20operator#command-operators)  to add %LOCAL_REPLY_BODY%

Reply:
Done